tests/network/probes.go

```diff
@@ -84,21 +83,19 @@ var _ = SIGDescribe("[ref_id:1182]Probes", func() {
 				By("Specifying a VMI with a readiness probe")
 				vmi = createReadyCirrosVMIWithReadinessProbe(virtClient, readinessProbe)
 
+				assertPodNotReady(virtClient, vmi)
+
+				// pod is not ready until our probe contacts the server
```

Comment:
comment position is pretty confusing now, 
also should it be VM ? 
and maybe can be updated to explain better (depends on the answer of the 2nd comment)